index.client.js

```diff
@@ -56,17 +56,28 @@ ReactI13nGoogleAnalytics.prototype.getPlugin = function () {
              * @param {Function} calback callback function
              */
             pageview: function (payload, callback) {
+                var args = {},
+                    argsKey = ['location', 'page', 'title'];
+
+                for (i in argsKey) {
+                    var prop = argsKey[i];
+                    if (payload.hasOwnProperty(prop)) {
```

Comment:
I think `payload[props]` is what you want, otherwise will still pick `undefined` value